docs/tech/application-ACL.md

```diff
@@ -118,50 +118,50 @@ message Transaction {
 
     bytes payload;
     bytes metadata;
-    
-    ...    
+
+    ... 
 }
 ```
 
-Another way to achieve this is to have the payload containing the metadata itself.  
+Another way to achieve this is to have the payload containing the metadata itself.
 
 ### Validators
 
-To assist chaincode execution, the validators provide the chaincode additional information, like the metadata and the binding.  
- 
+To assist chaincode execution, the validators provide the chaincode additional information, like the metadata and the binding.
+
 ## Application-level access control
- 
+
 ### Deploy Transaction
- 
-Alice has full control on the deployment transaction's metadata. 
-In particular, the metadata can be used to store a list of ACLs (one per function), or a list of roles. 
-To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been assigned 
-that privilege or role. The latter is done offline. 
 
-Now, Alice requires that to invoke the *hello* function, a certain message *M* has to be authenticated by an authorized invoker (Bob, in our case). 
+Alice has full control on the deployment transaction's metadata.
+In particular, the metadata can be used to store a list of ACLs (one per function), or a list of roles.
+To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been assigned
+that privilege or role. The latter is done offline.
+
+Now, Alice requires that to invoke the *hello* function, a certain message *M* has to be authenticated by an authorized invoker (Bob, in our case).
 We distinguish the following two cases:
 
 1. *M* is one of the chaincode's function arguments;
 2. *M* is the invocation message itself, i.e., function-name, arguments.
- 
+
 ### Execute Transaction
-  
+
 To invoke *hello*, Bob needs to sign *M* using the TCert/ECert Alice has used to name him in the deployment transaction's metadata.
 Let's call this certificate CertBob. At this point Bob does the following:   
 
 1. Bob gets a CertificateHandler for CertBob, *cHandlerBob*;
-2. Bob gets a new TransactionHandler to issue the execute transaction, *txHandler* relative to his next available TCert or his ECert; 
+2. Bob gets a new TransactionHandler to issue the execute transaction, *txHandler* relative to his next available TCert or his ECert;
```

Comment:
gets -> obtains

+ [ minor, but do we want some `TransactionHandler` style/formatting? minor, as in: really minor! ]